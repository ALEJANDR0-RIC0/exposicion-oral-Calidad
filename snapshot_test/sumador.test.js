const sumador = require('./sumador');

test('resultado del sumador coincide con snapshot', () => {
  const resultado = sumador(2, 3);

  // Para evitar fallos por el timestamp cambiante, lo eliminamos antes del snapshot
  delete resultado.timestamp;

  expect(resultado).toMatchSnapshot();
});