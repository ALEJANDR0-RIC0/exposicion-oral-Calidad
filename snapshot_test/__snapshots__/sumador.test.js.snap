// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`resultado del sumador coincide con snapshot 1`] = `
{
  "a": 2,
  "b": 3,
  "suma": 5,
}
`;
